src/traces/indicator/plot.js

```diff
@@ -0,0 +1,788 @@
+    ax._id = 'xbulletaxis';
+    ax.domain = [bulletLeft, bulletRight];
+    ax.setScale();
+
+    vals = Axes.calcTicks(ax);
+    transFn = Axes.makeTransFn(ax);
+    tickSign = Axes.getTickSigns(ax)[2];
+
+    shift = size.t + size.h;
+    if(ax.visible) {
+        Axes.drawTicks(gd, ax, {
+            vals: ax.ticks === 'inside' ? Axes.clipEnds(ax, vals) : vals,
+            layer: bulletaxis,
+            path: Axes.makeTickPath(ax, shift, tickSign),
+            transFn: transFn
+        });
+
+        Axes.drawLabels(gd, ax, {
+            vals: vals,
+            layer: bulletaxis,
+            transFn: transFn,
+            labelFns: Axes.makeLabelFns(ax, shift)
+        });
+    }
+
+    function drawRect(s) {
+        s
+            .attr('width', function(d) { return Math.max(0, ax.c2p(d.range[1] - d.range[0]));})
+            .attr('x', function(d) { return ax.c2p(d.range[0]);})
+            .attr('y', function(d) { return 0.5 * (1 - d.thickness) * bulletHeight;})
+            .attr('height', function(d) { return d.thickness * bulletHeight; });
+    }
+
+    // Draw bullet background, steps
+    var boxes = [gaugeBg].concat(trace.gauge.steps);
+    var targetBullet = bullet.selectAll('g.targetBullet').data(boxes);
+    targetBullet.enter().append('g').classed('targetBullet', true).append('rect');
+    targetBullet.select('rect')
+        .call(drawRect)
+        .call(styleShape);
+    targetBullet.exit().remove();
+
+    // Draw value bar with transitions
+    var fgBullet = bullet.selectAll('g.fgBullet').data([trace.gauge.value]);
+    fgBullet.enter().append('g').classed('fgBullet', true).append('rect');
+    fgBullet.select('rect')
+        .attr('height', innerBulletHeight)
+        .attr('y', (bulletHeight - innerBulletHeight) / 2)
+        .call(styleShape);
+    if(hasTransition) {
+        fgBullet.select('rect')
+            .transition()
+            .duration(transitionOpts.duration)
+            .ease(transitionOpts.easing)
+            .each('end', function() { onComplete && onComplete(); })
+            .each('interrupt', function() { onComplete && onComplete(); })
+            .attr('width', Math.max(0, ax.c2p(Math.min(trace.vmax, cd[0].y))));
+    } else {
+        fgBullet.select('rect')
+            .attr('width', Math.max(0, ax.c2p(Math.min(trace.vmax, cd[0].y))));
+    }
+    fgBullet.exit().remove();
+
+    var data = cd.filter(function() {return trace.gauge.threshold.value;});
+    var threshold = bullet.selectAll('g.threshold').data(data);
+    threshold.enter().append('g').classed('threshold', true).append('line');
+    threshold.select('line')
+        .attr('x1', ax.c2p(trace.gauge.threshold.value))
+        .attr('x2', ax.c2p(trace.gauge.threshold.value))
+        .attr('y1', (1 - trace.gauge.threshold.thickness) / 2 * bulletHeight)
+        .attr('y2', (1 - (1 - trace.gauge.threshold.thickness) / 2) * bulletHeight)
+        .style('stroke', trace.gauge.threshold.line.color)
```

Comment:
Please use `Color.stroke` 

https://github.com/plotly/plotly.js/blob/795220f815afcf79c73885715379b606afc1b4a1/src/components/color/index.js#L84-L87

whenever you need to set `stroke` for consistency and to guarantee that SVG exports of `indicator` traces will work in Illustrator.

Reply:
Done in e6d8525ab07c56ca10bdbac6762b06d1d57bf830